tests/cupyx_tests/jit_tests/test_cooperative_groups.py

```diff
@@ -78,12 +78,15 @@ def test_grid(x):
             g.sync()  # this should just work!
 
         x = cupy.empty((16,), dtype=cupy.uint64)
-        x[:] = -1  # = 2**64-1
+        x[:] = -1
         test_grid[2, 32](x)
         assert x[0] == 1
         assert x[1] == 64
         assert (x[2], x[3], x[4]) == (2, 1, 1)
-        assert (x[5:] == 2**64-1).all()
+        # TODO (ev-br): revert back to x[5:] == -1 after the edge case of
```

Comment:
```suggestion
        # XXX: np2.0: revert back to x[5:] == -1 after the edge case of
```
Let's make sure this is grep'able